lib/core/xskdev/xskdev.c

```diff
@@ -587,6 +592,35 @@ umem_create(lport_cfg_t *cfg)
     return NULL;
 }
 
+static void
+xskdev_get_channels_from_sysfs(const char *if_name, uint32_t *rxq, uint32_t *txq)
```

Comment:
This function looks like a duplicate to the one in the netdev_funcs.c can we use the netdev_funcs.c here?

Reply:
I will double check

Reply:
I moved xskdev.c to using the netdev funcs